### PR TITLE
[doc](mtmv)mtmv doc extend

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/query-acceleration/async-materialized-view/async-materialized-view.md [hunk 1]
```diff
@@ -207,3 +207,8 @@ DROP MATERIALIZED VIEW mv1;
 
 请参阅 [查询异步物化视图](./query-async-materialized-view.md)
 
+## 注意事项
+
+- 异步物化视图仅支持在Nereids优化器使用，[Nereids优化器](../nereids.md)
+- 当前判断物化视图和基表是否同步仅支持`OlapTable`， 对于其它外表，会直接认为是同步的，例如物化视图的基表全是外表，在查询`mv_infos()`时，SyncWithBaseTables会永远为1(true),在刷新物化视图时需要手动刷新指定的分区或指定`complete`刷新全部分区
```
Comment (morrySnow) [inline docs/zh-CN/docs/query-acceleration/async-materialized-view/async-materialized-view.md:213]:
```suggestion
- 当前判断物化视图和基表是否同步仅支持`OlapTable`.对于其它外表，会直接认为是同步的.例如，物化视图的基表全是外表。在查询`mv_infos()`时，SyncWithBaseTables会永远为1（true）。在刷新物化视图时需要手动刷新指定的分区或指定`complete`刷新全部分区
```

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-ASYNC-MATERIALIZED-VIEW.md [hunk 2]
```diff
@@ -170,11 +170,13 @@ KEY(k1,k2)
 
 物化视图特有的property包括：
 
-`grace_period`：查询改写时允许物化视图数据的最大延迟时间
+`grace_period`：查询改写时允许物化视图数据的最大延迟时间（单位：秒）,如果分区A和基表的数据不一致，物化视图的分区A上次刷新时间为1，系统当前时间为2，那么该分区不会被透明改写，但是如果grace_period大于等于1，该分区就会被用于透明改写
```
Comment (morrySnow) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-ASYNC-MATERIALIZED-VIEW.md:173]:
```suggestion
`grace_period`：查询改写时允许物化视图数据的最大延迟时间（单位：秒）。如果分区A和基表的数据不一致，物化视图的分区A上次刷新时间为1，系统当前时间为2，那么该分区不会被透明改写。但是如果grace_period大于等于1，该分区就会被用于透明改写
```

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-ASYNC-MATERIALIZED-VIEW.md [hunk 3]
```diff
@@ -170,11 +170,13 @@ KEY(k1,k2)
...
 
 `excluded_trigger_tables`：数据刷新时忽略的表名，逗号分割。例如`table1,table2`
 
-`refresh_partition_num`：单次insert语句刷新的分区数量，默认为1
+`refresh_partition_num`：单次insert语句刷新的分区数量，默认为1, 物化视图刷新时会先计算要刷新的分区列表，然后根据该配置拆分成多个insert语句顺序执行，遇到失败的insert语句，整个任务将停止执行，物化视图保证单个insert语句的事务性，失败的insert语句不会影响到已经刷新成功的分区。
```
Comment (morrySnow) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-ASYNC-MATERIALIZED-VIEW.md:177]:
```suggestion
`refresh_partition_num`：单次insert语句刷新的分区数量，默认为1。物化视图刷新时会先计算要刷新的分区列表，然后根据该配置拆分成多个insert语句顺序执行。遇到失败的insert语句，整个任务将停止执行。物化视图保证单个insert语句的事务性，失败的insert语句不会影响到已经刷新成功的分区。
```

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-ASYNC-MATERIALIZED-VIEW.md [hunk 4]
```diff
@@ -170,11 +170,13 @@ KEY(k1,k2)
...
+
+`workload_group`：物化视图执行刷新任务时使用的workload_group名称，用来限制物化视图刷新数据使用的资源，避免影响到其它业务的运行。workload_group创建及使用 [WORKLOAD-GROUP](../../../../admin-manual/workload-group.md)
```
Comment (morrySnow) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-ASYNC-MATERIALIZED-VIEW.md:179]:
```suggestion
`workload_group`：物化视图执行刷新任务时使用的workload_group名称。用来限制物化视图刷新数据使用的资源，避免影响到其它业务的运行。workload_group创建及使用 [WORKLOAD-GROUP](../../../../admin-manual/workload-group.md)
```

**Comment by zddr** [comment]
run buildall

File: docs/en/docs/query-acceleration/async-materialized-view/async-materialized-view.md [hunk 5]
```diff
@@ -206,3 +206,8 @@ Specific syntax can be viewed [DROP MATERIALIZED VIEW](../../sql-manual/sql-refe
 ## The use of materialized views
 
 can be viewed [Query async materialized view](./query-async-materialized-view.md)
+
+## Notice
+
+- Asynchronous materialized views are only supported for use in the Nereids optimizer, [Nereids optimizer](../nereids.md)
+- Currently, only `OlapTable` is supported for determining whether the materialized view and base table are synchronized. For other appearances, they are directly considered synchronous. For example, the base tables of materialized views are all appearances. When querying `mv_infos()`, SyncWithBaseTables will always be 1 (true). When refreshing the materialized view, it is necessary to manually refresh the specified partition or specify `complete` to refresh all partitions
```
Comment (morrySnow) [inline docs/en/docs/query-acceleration/async-materialized-view/async-materialized-view.md:213]: determining -> judging. appearances -> external table? You can use products like Chat GPT to obtain more accurate translation results.

**Comment by zddr** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by zy-kkk — APPROVED** [review]

**Pull request merged**